app/scripts/controllers/overview.js

```diff
@@ -427,6 +426,11 @@ angular.module('openshiftConsole')
           Logger.log("routes (subscribe)", $scope.routesByService);
         }));
 
+        watches.push(DataService.watch("endpoints", context, function(endpointsData) {
```

Comment:
Do you know if endpoints will show up for pods that are Running but whose containers are not Ready?